tests/acceptance/test_simplified_long_covid_study.py

```diff
@@ -99,12 +99,24 @@ def test_simplified_cohort(database, setup_tpp_database):
             event(code="1325161000000102", date="2020-10-10"),  # post-covid syndrome
         ),
         # excluded by registration date
-        *patient(2, "M", registration(start_date="2001-01-01", end_date="2002-02-02")),
+        *patient(
+            2,
+            "M",
+            "1990-1-1",
+            registration(start_date="2001-01-01", end_date="2002-02-02"),
+        ),
     )
+    # try:
+    #     res = extract(SimplifiedCohort, TPPBackend, database)
+    # except Exception as e:
+    #     import ipdb; ipdb.set_trace()
+    #     pass
+
```

Comment:
I think this was probably left in by mistake.

Reply:
ah, yes :facepalm: